app/mailers/candidate_mailer.rb

```diff
@@ -60,15 +60,17 @@ def new_interview(application_choice, interview)
     )
   end
 
-  def interview_updated(application_choice, interview)
+  def interview_updated(application_choice, interview, old_course)
     @application_form = application_choice.application_form
     @interview = interview
     @provider_name = interview.provider.name
     @course_name_and_code = application_choice.current_course_option.course.name_and_code
+    @old_course_name_and_code = old_course.course.name_and_code
+    @course_has_changed = @course_name_and_code != @old_course_name_and_code
```

Comment:
This appears to be repeated in the mailer template.
Would inverting the logic to populate a variable like `@updated_course_name_and_code` be simpler?
If this is nil we can shortcut like

```ruby
course_name_and_code: @updated_course_name_and_code || @course_name_and_code
```
and in the template:

```ruby
The details of your interview for <%= @updated_course_name_and_code || @course_name_and_code %> have been updated.
```
also 

```ruby
<% if @updated_course_name_and_code %>
  The interview is with <%= @provider_name %>. It is now for <%= @updated_course_name_and_code %>.
...
```

Slightly less ternary logic and less instance vars.

Reply:
I've removed ternary logic and reworked the method based on these suggestions. However, these wouldn't quite work? E.g if the `@updated_course_name_and_code` was true, it would return true as the value instead of a course name & code

Reply:
`@updated_course_and_name_code` would be nil or the name and code of a course. Not a boolean. The || shortcut operator would trigger on it being a nil value.

Reply:
not sure I follow, `@updated_course_and_name_code` would be the `current_course_option` which couldn't be nil whether the course was updated or not

Reply:
fixed in 34e7702